packages/types/src/validation.ts

```diff
@@ -94,9 +95,9 @@ export const validationResultSchema = z
                 .regex(hexPattern)
                 .transform((val) => val as HexData)
         }),
-        stakeInfoSchema,
-        stakeInfoSchema,
-        stakeInfoSchema
+        stakeInfoSchema.optional(),
```

Comment:
is it actually optional? i thought it's always there

Reply:
this is factory & paymaster info. this should be optional because we may or may not have paymaster & initcode in userop.